src/jpeg-turbo.lisp

```diff
@@ -12,8 +12,10 @@
 
 ;; Library
 (define-foreign-library jpeg-turbo
+  (:os-macosx (:or "libturbojpeg.0.dylib" "libturbojpeg.dylib"))
```

Comment:
Depend on [cffi doc](https://common-lisp.net/project/cffi/manual/html_node/define_002dforeign_002dlibrary.html), `load the library name in order for the first one where feature evaluates to true`, so I put `os-macosx` before `unix`.